draft-ietf-rats-corim.md

```diff
@@ -1368,12 +1368,16 @@ The Conditional Endorsement Series Triple has the following structure:
 The `conditional-endorsement-series-triple-record` has the following parameters:
 
 * `condition`: Search criteria that locates Evidence, corroborated Evidence, or Endorsements.
+The condition is a record containing an `environment`, optional `claims-list`, and optional `authorized-by`.
+If both `authorized-by` and `claims-list.authorized-by` are populated then `claims-list.authorized-by` is ignored.
```

Comment:
```suggestion
If both `authorized-by` and `claims-list`.`authorized-by` are populated then `claims-list`.`authorized-by` is ignored.
```
I think we weren't using code-face for the projection syntax since it's hypothetical.